app/uk/gov/hmrc/eoricommoncomponent/frontend/views/partials/styles.scala.html

```diff
@@ -20,14 +20,12 @@
 @this(head_di: Head)
 @()
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<!--[if gt IE 8]><!--> <link rel="stylesheet" media="screen" href="@routes.Assets.versioned("template/govuk-template.css")" type="text/css"> <!--<![endif]-->
 
 
 @head_di(
     linkElem = None,
     headScripts = None)
 
-<link rel="stylesheet" href="@routes.Assets.versioned("template/govuk-template-print.css")" media="print" type="text/css" />
```

Comment:
I tried to keep those two lines, because I'm not sure if we need to load it or not. Govuk-frontend doesn't have this in the instructions to setup it